src/Microsoft.DocAsCode.Plugins/IHostService.cs

```diff
@@ -7,7 +7,8 @@ namespace Microsoft.DocAsCode.Plugins
 
     public interface IHostService
     {
-        MarkupResult Markup(string markdown, FileAndType ft);
+        string MarkupToHtml(string markdown, string file);
+        MarkupResult Markup(string markdown, FileAndType ft, bool isMarkuped = false);
```

Comment:
Modify parameter is a breaking change, all plugin require recompile.
And `string markdown` is unclear when isMarkuped is true.
So just add a new method, do html -> MarkupResult?